core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java

```diff
@@ -84,6 +84,8 @@
     private final AsciiString scheme;
     private final InboundTrafficController inboundTrafficController;
     private final ServerHttp1ObjectEncoder writer;
+    @Nullable
+    private final KeepAliveHandler keepAliveHandler;
```

Comment:
Perhaps we can keep accessing it via `writer.keepAliveHandler()`, instead of adding a new member field?

Reply:
Fixed